vocabularies/UI.xml

```diff
@@ -1018,6 +1023,11 @@ The trend is
         <Property Name="Url" Type="Edm.String" Nullable="false">
           <Annotation Term="Core.IsURL" />
           <Annotation Term="Core.Description" String="URL of referenced information" />
+          <Annotation Term="Validation.ApplicableTerms">
```

Comment:
what should happen to the facet if you annotate it with "LinkTarget" - how should the UI behave?

Reply:
When the UI contains a link to that URL, the link shall open according to the `HTML5.LinkTarget`. In other words: The `sap.m.Link` element shall get a corresponding `target` property.